client/quantum_serverless/core/decorators.py

```diff
@@ -64,7 +72,11 @@ def from_dict(cls, dictionary: dict):
         return Target(**dictionary)
 
 
-def run_qiskit_remote(target: Optional[Union[Dict[str, Any], Target]] = None):
+def run_qiskit_remote(
+    target: Optional[Union[Dict[str, Any], Target]] = None,
+    state: Optional[StateHandler] = None,
+    events_handler: Optional[EventHandler] = None,
```

Comment:
Can we rename the events_handler by just events? to be consistent with state interface as well